include/crypto/x509v3.h

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2020 The OpenSSL Project Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License 2.0 (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+
+#pragma once
```

Comment:
Probably should have guards like this:

````
#ifndef OSSL_CRYPTO_X509V3_H
# define OSSL_CRYPTO_X509V3_H
````
At least that's what we do in (some) of these other internal include files.

Reply:
#pragma once is the modern method than the guards
But if yoy want i will do...
**Fixed**